AppCenter/AppCenter/MSAppCenterErrors.h

```diff
@@ -1,29 +1,31 @@
 #import <Foundation/Foundation.h>
 
+#define MS_APP_CENTER_BASE_DOMAIN @"com.Microsoft.AppCenter."
+
 NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - Domain
 
-extern NSString *const kMSACErrorDomain;
+static NSString *const kMSACErrorDomain = MS_APP_CENTER_BASE_DOMAIN @"ErrorDomain";
 
 #pragma mark - Log
 
 // Error codes
 NS_ENUM(NSInteger){kMSACLogInvalidContainerErrorCode = 1};
 
 // Error descriptions
-extern NSString const *kMSACLogInvalidContainerErrorDesc;
+static NSString const *kMSACLogInvalidContainerErrorDesc = @"Invalid log container";
 
 #pragma mark - Connection
 
 // Error codes
 NS_ENUM(NSInteger){kMSACConnectionPausedErrorCode = 100, kMSACConnectionHttpErrorCode = 101};
 
 // Error descriptions
-extern NSString const *kMSACConnectionHttpErrorDesc;
-extern NSString const *kMSACConnectionPausedErrorDesc;
+static NSString const *kMSACConnectionHttpErrorDesc = @"An HTTP error occured.";
+static NSString const *kMSACConnectionPausedErrorDesc = @"Cancelled, connection paused with log deletion.";
```

Comment:
"Cancelled" is technically the British English spelling, and "Canceled" is the American English spelling... Both are correct, but we should default to American English spellings

```suggestion
static NSString const *kMSACConnectionPausedErrorDesc = @"Canceled, connection paused with log deletion.";
```

I only learned about this recently too 😄

Reply:
https://www.grammarly.com/blog/canceled-vs-cancelled/ for reference